src/10_functions.py

```diff
@@ -1,12 +1,25 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
-# YOUR CODE HERE
+
+def is_even(n):
+    if n % 2 == 0:
+        return True
+    else:
+        return False
```

Comment:
A coding tip I got from an interview once, you can just shortcut this into `return n % 2 == 0`... no need for an if/else.